cliStocksTracker.py

```diff
@@ -298,36 +247,36 @@ def average_buyin(self, buys: list, sells: list):
 
         return count, bought_at
 
+    # download all ticker data in a single request
+    # harder to parse but this provides a signficant performance boost
+    def download_market_data(self, args, stocks):
+        # get graph time interval and period
+        time_period = args.time_period if args.time_period else "1d"
+        time_interval = args.time_interval if args.time_interval else "1m"
+
+        try:
+            return market.download(
+                tickers=stocks,
+                period=time_period,
+                interval=time_interval,
+                progress=False,
+            )
+        except:
+            print(
+                "cliStocksTracker must be connected to the internet to function. Please ensure that you are connected to the internet and try again."
+            )
```

Comment:
single download for all ticker data - also wrapped in the try catch from a previous PR